pkg/networkplugin-syncer/main.go

```diff
@@ -26,7 +26,13 @@ func main() {
 	// set up signals so we handle the first shutdown signal gracefully
 	stopCh := signals.SetupSignalHandler()
 
-	registry := event.NewRegistry("networkplugin-syncer", os.Getenv("NETWORK_PLUGIN"))
+	networkPlugin := os.Getenv("SUBMARINER_NETWORKPLUGIN")
```

Comment:
As a side note, we can make this `NETWORK_PLUGIN` is the operator. There was a recent discussion about `NODE_NAME` and whether to use the `SUBMARINER_`.  We should be consistent. @sridhargaddam

Reply:
it's fine as SUBMARINER_NETWORKPLUGIN too, it's the network plugin we are instantiating in the different submariner components, 

I'd leave it as it is for now give the time restrictions we have.

Reply:
W.r.t NODE_NAME, I wanted to avoid SUBMARINER as its not something related to SUBMARINER but is OS/Host information. Regarding NETWORK_PLUGIN, I'm fine with anything. AFAIK, it will not be parsed along (or at the same place) where other SUBMARINER_* env variables are parsed/used. So, using NETWORK_PLUGIN might make more sense.